service_crategen/Cargo.toml

```diff
@@ -31,6 +31,12 @@ env_logger = "0.4.0"
 optional = true
 version = "0.0"
 
+[profile.dev]
+opt-level = 1
+codegen-units = 2
+debug = false
```

Comment:
We don't currently use these but they are a bit of a landmine for future profiling as it may be expected the have the defaults in this compilation profile.  Add a note in the README about this?